### DISCUSSION TITLE
gradient dithering

### DISCUSSION THREAD

**Discussion by GottZ** [zen-browser/desktop#932] [Ideas]
### What happened?

this is primarily a feature request, adding something not even upstream firefox has, even though the impact is undoubtedly minimal.

[original reddit post from five years ago](https://www.reddit.com/r/web_design/comments/bnrezy/will_gradients_in_firefox_ever_become_pretty/)

![screenshot comparing edge to zen](https://home.gottz.de/gradients-firefox.png)

[link for testing](https://codepen.io/GottZ/pen/JqKmVx)

firefox sadly never applied dithering to radial and linear css gradients. I do remember them having it as feature flag long ago when they started implementing their rust based renderer.
it was removed after tho.
there is a good chance traces of it still remain.
I just have no clue where to look.

as this is literally the only thing preventing me from using anything firefox based, I would love to see dithering added in zen.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.29 (64-bit)

### Severity impact

Low

### What platform are you seeing the problem on?

Linux

**Comment by Anthonyy232** [comment]
This would be awesome!

**Comment by nyabinary** [comment]
https://www.firefox.com/en-US/firefox/144.0beta/releasenotes/
Good news :P upstream Firefox has dithering now
